diff --git a/node_modules/npm/node_modules/pacote/lib/util/git.js b/node_modules/npm/node_modules/pacote/lib/util/git.js
index 7642eb2..7bb3324 100644
--- a/node_modules/npm/node_modules/pacote/lib/util/git.js
+++ b/node_modules/npm/node_modules/pacote/lib/util/git.js
@@ -25,7 +25,8 @@ const GOOD_ENV_VARS = new Set([
   'GIT_SSH',
   'GIT_SSH_COMMAND',
   'GIT_SSL_CAINFO',
-  'GIT_SSL_NO_VERIFY'
+  'GIT_SSL_NO_VERIFY',
+  'GIT_CONFIG_GLOBAL'
 ])
 
 const GIT_TRANSIENT_ERRORS = [
